src/css/modules/_PageBreadcrumbs.scss

```diff
@@ -53,3 +53,31 @@
     }
   }
 }
+
+.rtl {
+  .coa-PageBreadcrumbs {
+    direction: rtl;
+    display: flex;
```

Comment:
all the base styles that could work for left to right or right to left -- can we move them up to
`.coa-PageBreadcrumbs` so the component style behavior works the same for both rtl and ltr -- and it's obvious what is changed specifically for rtl.

so, can you move these two up to the generic .coa-PageBreadcrumbs
`display: flex;
justify-content: center;
`

Reply:
Also -- this might be dumb -- but why both row-reverse and direction rtl?

Reply:
So I think I got confused here on the styling because the breadcrumbs aren't pulling in the arabic translations yet which makes them act weird. Once those are coming in, I think the ordering will behave better, but we might need some refactoring. I'll log an issue.

Reply:
👍